console-ui/src/pages/ServiceManagement/ServiceDetail/EditServiceDialog.js

```diff
@@ -129,9 +133,25 @@ class EditServiceDialog extends React.Component {
     wrapperCol: { span: 14 },
   });
 
+  getSelectorTypes() {
+    request({
+        method: 'GET',
+        url: 'v1/ns/service/selector/types',
```

Comment:
Should we use `v2` link?

Reply:
@KomachiSion Now all the API in `naming` starting with `v1`, due to this api is in `com.alibaba.nacos.naming.controllers.ServiceController`, so the api keeps starting with `v1`, if need change to `v2`. I need to move this api a now controller. I think we can keep the prefix `v1` now, and change all the apis in `v2` when we have an upgrade. how do you think about this?

Reply:
ok